sos/plugins/ceph.py

```diff
@@ -50,28 +50,42 @@ def setup(self):
         ])
 
         self.add_cmd_output([
-            "ceph status",
-            "ceph health detail",
-            "ceph osd tree",
-            "ceph osd stat",
-            "ceph osd dump",
             "ceph mon stat",
             "ceph mon_status",
             "ceph quorum_status",
-            "ceph mon dump",
-            "ceph df",
+            "ceph osd erasure-code-profile ls",
             "ceph report",
-            "ceph osd df tree",
-            "ceph fs dump --format json-pretty",
-            "ceph fs ls",
-            "ceph pg dump",
-            "ceph health detail --format json-pretty",
             "ceph osd crush show-tunables",
             "ceph-disk list",
             "ceph versions",
             "ceph osd crush dump"
+            "ceph -v"
```

Comment:
Add a coma to the end of previous line (that's the "implicit concatenation" failure of LGTM - currently a command `"ceph osd crush dumpceph -v"` would be executed).